src/containers/Token/index.jsx

```diff
@@ -35,16 +36,21 @@ ERROR_MESSAGES.default = {
 const getErrorMessage = (error) =>
   ERROR_MESSAGES[error] || ERROR_MESSAGES.default
 
-class Token extends Component {
-  static getDerivedStateFromProps(nextProps, prevState) {
-    const { match } = nextProps
-    return {
-      accountId: match.params.id,
-      prevId: prevState && prevState.accountId,
+const Token = ({ error }) => {
+  const { currency, id: accountId } = useParams()
+  const { t } = useTranslation()
+  const showError = error
+
+  useEffect(() => {
+    document.title = `${t('xrpl_explorer')} | ${accountId.substr(0, 12)}...`
```

Comment:
Why not use helmet?

Reply:
This is how it is done throughout the app.  I will create a ticket to update this to helmet everywhere.

Reply:
#650